labelImg.py

```diff
@@ -1428,12 +1428,15 @@ def close_file(self, _value=False):
     def delete_image(self):
         delete_path = self.file_path
         if delete_path is not None:
-            self.open_next_image()
-            self.cur_img_idx -= 1
+            del self.m_img_list[self.cur_img_idx]
+            self.file_list_widget.takeItem(self.cur_img_idx)
+
+            self.cur_img_idx -= 1  # self.open_next_img will increment idx.
```

Comment:
Removal of a path caused left shift of all of the following paths and the next image's path has the same index as deleted one.
Decrementing because  `self.open_next_img` will increment index later.